foundations/tying_it_all_together/conclusion.md

```diff
@@ -56,4 +56,4 @@ The journey has just begun!
 
 This section contains helpful links to related content. It isn’t required, so consider it supplemental.
 
-- It looks like this lesson doesn't have any additional resources yet. Help us expand this section by contributing to our curriculum.
+* If you are still confuse, Checkout this article to get more facts, prons and cons about both languages [Cloud-Infrastucter- service article](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).
```

Comment:
Can we correct the typos here please.

 If you are still confused, Checkout this article to get more facts, pros and cons about both languages [Ruby on Rails vs Javascript](https://cloudinfrastructureservices.co.uk/ruby-on-rails-vs-javascript-whats-the-difference/).